### PR TITLE
generic text added

### PR DESCRIPTION
**Pull request by shilpavij**
Generic text added based on the feedback https://github.com/SAPDocuments/Tutorials/issues/5169

### CODE REVIEW

File: tutorials/cp-apim-shoppingfresh-inspect-apiproxy/cp-apim-shoppingfresh-inspect-apiproxy.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ primary_tag: products>sap-api-management
     ![Service Callout](05-servicecallout-geocode.png)
 
     >Note the `app_id` and `app_code` variables are hard coded for the scope of this exercise. We could have even retrieved these values from a secure key value map store.
+    >If you are using the newer HERE trial `app_id` and `app_code` is combined into `appKey`. Kindly keep a watch.
```
Comment (MichaelCzcz) [inline tutorials/cp-apim-shoppingfresh-inspect-apiproxy/cp-apim-shoppingfresh-inspect-apiproxy.md:45]: Please clarify: "Kindly keep a watch"

**Comment by thecodester** [comment]
@shilpavij Can you respond to Michael's comment?

**Comment by shilpavij** [comment]
Done.

Regards
Shilpa

**Comment by shilpavij** [comment]
Dear team,

I can not see the update in the tutorial. It still reflects the old one. Kindly check

Regards
Shilpa

From: Vij, Shilpa
Sent: Tuesday, May 19, 2020 9:30 AM
To: SAPDocuments/Tutorials <reply@reply.github.com>; SAPDocuments/Tutorials <Tutorials@noreply.github.com>
Cc: Mention <mention@noreply.github.com>
Subject: RE: [SAPDocuments/Tutorials] generic text added (#5171)

Done.

Regards
Shilpa

**Comment by shilpavij** [comment]
Specifically mentioned CRM Hub for Hubspot

**Pull request merged**